contracts/standard/ExampleToken.cdc

```diff
@@ -1,38 +1,49 @@
 import "FungibleToken"
 import "MetadataViews"
+import "FungibleTokenMetadataViews"
 
-pub contract ExampleToken: FungibleToken {
+access(all) contract ExampleToken: FungibleToken {
 
     /// Total supply of ExampleTokens in existence
-    pub var totalSupply: UFix64
-    
-    /// Storage and Public Paths
-    pub let VaultStoragePath: StoragePath
-    pub let VaultPublicPath: PublicPath
-    pub let ReceiverPublicPath: PublicPath
-    pub let AdminStoragePath: StoragePath
+    access(all) var totalSupply: UFix64
 
+    /// TokensInitialized
+    ///
     /// The event that is emitted when the contract is created
-    pub event TokensInitialized(initialSupply: UFix64)
+    access(all) event TokensInitialized(initialSupply: UFix64)
 
+    /// TokensWithdrawn
+    ///
     /// The event that is emitted when tokens are withdrawn from a Vault
-    pub event TokensWithdrawn(amount: UFix64, from: Address?)
+    access(all) event TokensWithdrawn(amount: UFix64, from: Address?)
```

Comment:
FT emit standard events now, do we need this to emit its own?

Reply:
Personally I am going for the least disruptive changes required for this contract to be ready. On that basis, what a standard contract that's used just for testing does wasn't really a concern to me

I personally would rather update these at a later date since they don't have much to do with how HybridCustody itself will operate